presto-parser/src/main/java/com/facebook/presto/sql/tree/FunctionCall.java

```diff
@@ -24,50 +24,58 @@
 {
     private final QualifiedName name;
     private final Optional<Window> window;
+    private final Optional<FilterClause> filterClause;
```

Comment:
Instead of a new `FilterClause` class, I would just use `Optional<Expression>` here